migration/migration.go

```diff
@@ -46,26 +36,12 @@ func (f MigrationFunc) Migrate(_ context.Context, txn db.Transaction, network *u
 // Before is a no-op.
 func (f MigrationFunc) Before(_ []byte) error { return nil }
 
+// todo comment
+const startMigrationVersion = 17
```

Comment:
Would it be a good idea to start from 0 again? There are a few other migrations that need to be merged soon. Maybe it is a good idea to merge it after that?

Reply:
> There are a few other migrations that need to be merged soon. Maybe it is a good idea to merge it after that?

You mean merge this PR after all migrations in `main` ? Sure.

> Would it be a good idea to start from 0 again?
In this case we need to do something like this:
```
if migration.Version > 0 {
    if migration.Version < 17 {
         return err
    } else migration.Version == 17 { 
        migration.Version = 0
    } else {
        // above 17 can't be ?
    }
}
```

Also when we apply migrations there is "progress bar" for migrations:
```
Applying migration 16/17
```
if you think it doesn't worth to maintain sequential order even if we remove obsolete migrations (I'm ok with that) - let's do that